docs/automobile/openstreetmap-importer.md

```diff
@@ -50,10 +50,23 @@ Type in the [Windows Command Prompt](https://en.wikipedia.org/wiki/Cmd.exe) open
 
 You should use the `importer.py` Python script to generate the `myMap.wbt` webots simulation world from the `myMap.osm` file:
 
+%tab-component "os"
+
+%tab "Windows & Linux"
```

Comment:
Because the tabs are memorized for other pages, we shouldn't create a tab entitled "Windows & Linux". Instead we should a make a "Windows" tab and a "Linux" tab with the same content (unfortunately).